### ISSUE TITLE
Exercise 14-1: Focus ring with good contrast

### ISSUE THREAD

**Issue by braheezy** [braheezy/zibra#2]
Source: book/book/accessibility.md

Exercise 14-1

*Focus ring with good contrast*. Improve the contrast of the focus
indicator by using two outlines, a thicker white one and a thinner
black one, to ensure that there is contrast between the focus ring and
surrounding content.
